src/request/notification/localized.rs

```diff
@@ -84,8 +87,8 @@ impl<'a> LocalizedNotificationBuilder<'a> {
     {
         LocalizedNotificationBuilder {
             alert: LocalizedAlert {
-                title: title,
-                body: body,
+                title: Some(title),
```

Comment:
Are these actually optional fields? If so, they should be separate builder methods...